packages/fast-form-generator-react/src/form/form-item.textarea.style.ts

```diff
@@ -17,7 +17,9 @@ const styles: ComponentStyles<IFormItemTextareaClassNameContract, {}> = {
     formItemTextarea_textarea: {
         ...applyInputStyle(),
         marginTop: toPx(8),
-        marginBottom: toPx(2)
+        marginBottom: toPx(2),
+        resize: "none",
```

Comment:
Did you consider just restricting to vertical resize? I understand why we wouldn't want to do a horizontal resize as it doesn't align with the container but I could see how vertical resize could be nice if typing long paragraphs.

Reply:
I think part of the reason for removing here is that it has poor cross-browser support (no IE or Edge support at all). For a consistent experience we discussed simply removing this all up.

Reply:
My opinion is that we should keep vertical if its available - I don't think we should remove useful features just get consistency.

Reply:
I'm fine with this ^^^

Reply:
Just verified that `resize: "vertical"` works in firefox and chrome